.github/workflows/dev-stg-release.yml

```diff
@@ -81,8 +81,18 @@ jobs:
                       </mirrors>
 
                     </settings>' > ~/.m2/settings.xml
-        - name: Maven Build
```

Comment:
Shall we remove other maven related stuff like .m2 settings.xml stuff in line 83